messages/src/main/java/no/unit/nva/publication/messages/MessageApiConfig.java

```diff
@@ -4,7 +4,12 @@ public final class MessageApiConfig {
     
     public static final String MESSAGE_IDENTIFIER_PATH_PARAMETER = "messageIdentifier";
     public static final String MESSAGE_NOT_FOUND = "Could not find message";
+    public static final String MESSAGE_PATH = "message";
```

Comment:
duplication of certain constants in order to avoid importing the "ticket" module